tests/indent/java_spec.lua

```diff
@@ -21,5 +21,9 @@ describe("indent Java:", function()
     run:new_line("enum.java", { on_line = 2, text = "THING_B,", indent = 2 })
     run:new_line("class_with_annotation.java", { on_line = 2, text = "void foo() {}", indent = 2 })
     run:new_line("enum_with_annotation.java", { on_line = 2, text = "THING;", indent = 2 })
+    run:new_line("interface.java", { on_line = 1, text = "void foo();", indent = 2 })
+    run:new_line("javadoc.java", { on_line = 2, text = "* Sample javadoc line", indent = 3 })
+    --TODO: Fix indentation when adding line *before* the ERROR line.
```

Comment:
Can you please add this as expected failure test? Then it wi be visible and also error when it gets (accidentally) fixed. See other languages that have XFAIL tests

Reply:
Added an XFAIL test for this.